src/clj/cuic/chrome.clj

```diff
@@ -346,20 +353,23 @@
          data-dir (or custom-data-dir tmp-data-dir)
          port (or (:remote-debugging-port options)
                   (get-free-port-for-cdp))
-         args (->> (merge (if (:headless options)
-                            headless-defaults
-                            defaults)
-                          options
-                          {:user-data-dir         (str data-dir)
-                           :remote-debugging-port port})
-                   (filter second)
+         options (merge (resolve-headless-opts options)
```

Comment:
I think `resolve-headless-opts` is a little bit misleading naming here since the function applies defaults if the `:headless` flag is set to `false`. I'd either

1. separate headless handling out of function
```clj
(if-some [headless-mode (:headless options)]
  (resolve-headless-defaults headless-mode)
  defaults)
```

2. or rename the function to e.g. `(resolve-defaults options)`

Reply:
Good catch, That naming was indeed confusing!